handbook/communication/content_guidelines/style_and_mechanics.md

```diff
@@ -743,19 +748,19 @@ Don’t capitalize descriptive product or feature names. Features are not proper
 </div>
 </div>
 
-Use the natural plural/singular form. If a feature name is a plural noun, treat it as a plural noun.
+Use the natural plural/singular form when describing the functionality. 
```

Comment:
Do you have an example sentence in which "Batch Changes" would describe the _functionality_  and be plural?

Reply:
"The batch changes I created yesterday are already complete."